string name       # Name of the waypoint, SAVE is reserved for saving command
---
bool success         # Did the operation succeed?